recipe/build.sh

```diff
@@ -8,21 +8,41 @@ if [[ "$(echo $fortran_compiler_version | cut -d '.' -f 1)" -gt 9 ]]; then
   export FFLAGS="$FFLAGS -fallow-argument-mismatch"
 fi
 
+if [[ ${HOST} =~ .*linux.* ]]; then
+  # Need to point to libquadmath.so.0
+  export LD_LIBRARY_PATH=${PREFIX}/lib:$LD_LIBRARY_PATH
+fi
+
+if [[ ${HOST} =~ .*darwin.* ]]; then
+  export LIBS="$LIBS -framework Accelerate"
+  export FFLAGS="$FFLAGS -ff2c -fno-second-underscore"
+  export FCFLAGS="$FCFLAGS -ff2c -fno-second-underscore"
+fi
+
 for shared_libs in OFF ON
 do
   cmake ${CMAKE_ARGS} \
     -DCMAKE_PREFIX_PATH=${PREFIX} \
     -DCMAKE_INSTALL_PREFIX=${PREFIX} \
     -DCMAKE_INSTALL_LIBDIR=lib \
     -DBUILD_SHARED_LIBS=${shared_libs} \
-    -DLAPACK_LIBRARIES="-llapack" \
-    -DBLAS_LIBRARIES="-lblas" \
     -DICB=ON \
     -DMPI=${DMPI} \
+    -DBLAS_LIBRARIES="-lopenblas" \
```

Comment:
Do we need `-DEIGEN=ON -DPYTHON3=ON` too? See https://github.com/opencollab/arpack-ng/blob/6d625dbf34375a0b5823d6c03bd16044f405baae/.github/workflows/jobs.yml#LL132C49-L132C72

Reply:
To allow python3 support through -DPYTHON3 flag, arpack asks for boost v 1.78 but our latest is 1.73.

Reply:
Ah, thanks

Reply:
I have added EIGEN support.